qiskit/optimization/converters/inequality_to_equality.py

```diff
@@ -0,0 +1,361 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""The inequality to equality converter."""
+
+import copy
+import math
+from typing import List, Tuple, Dict, Optional
+import logging
+
+from ..problems.quadratic_program import QuadraticProgram
+from ..problems.quadratic_objective import QuadraticObjective
+from ..problems.constraint import Constraint
+from ..problems.variable import Variable
+from ..exceptions import QiskitOptimizationError
+
+logger = logging.getLogger(__name__)
+
+
+class InequalityToEquality:
+    """Convert inequality constraints into equality constraints by introducing slack variables.
+
+    Examples:
+        >>> problem = QuadraticProgram()
+        >>> # define a problem
+        >>> conv = InequalityToEquality()
+        >>> problem2 = conv.encode(problem)
+    """
+
+    _delimiter = '@'  # users are supposed not to use this character in variable names
+
+    def __init__(self) -> None:
+        """Initialize the inequality to equality variable converter."""
```

Comment:
This comment does not add any value that the programming language itself does not tell you. I would just delete it